src/ui/profile_preferences.ui

```diff
@@ -1014,7 +1017,21 @@ you can use it but there could be issues with aligning columns of text</string>
                 </size>
                </property>
                <property name="text">
-                <string>Wrap lines at:</string>
+                <string>Wrap lines:</string>
+               </property>
+              </widget>
+             </item>
+             <item>
+              <widget class="QCheckBox" name="checkBox_autoWrap">
+               <property name="text">
+                <string>automatically</string>
+               </property>
+              </widget>
+             </item>
+             <item>
+              <widget class="QLabel" name="label_6">
+               <property name="text">
+                <string>or at</string>
```

Comment:
Please find a way to combine these strings into one:
- Wrap lines
- automatically
- or at

Because they will be translated into multiple languages that may or may not follow a same grammar